src/data/contributors/Rahul.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Rahul Kumar",
+    "skills": ["MERN", "Python", "Machine Learning", "Android"],
+    "cause": "the excitement",
+    "image": "Avatar5",
+    "github": "https://github.com/Rahul-D78",
+    "twitter": "https://twitter.com/",
```

Comment:
provide your full id here if not leave blank

Reply:
@Rahul-D78 ping!

Reply:
Hey @Abhishek-kumar09

Here is his full id link: https://twitter.com/RahulKu36162566

You can do the same thing which I suggested few days back.

Reply:
Thanks @Suyash-Purwar for your review, we had created this repo so that everyone old or new get some kick of how git and github contributions work, because approaching big projects in first step is overwhelming. 
Students could get over the barrier of commiting changes and starting with git, was our primary goal.
You have had really done a good research to find out twitter and linkedIn ID of different peaple, but still that doesn't serve the purpose until the contributor itself play with those things.

So really thankful to you first, and merging this PR.

Reply:
Firstly, I am happy to help! 😀 
Secondly, I agree with that certainly. If I were you, maybe I would have deleted the old PRs.

Looking forward to making some more meaningful contributions.

Reply:
@Suyash-Purwar Do you wanted to work with node project, we have two open projects for that, You can tell me if you know react too, I can help you with getting started with some beginner friendly issues.

Reply:
@Suyash-Purwar Sure, I would love to be a part of it! I used to code extensively but I was on a break for one year. I know Node.js, Express and, MongoDB and looking forward to learning React.js soon.

I can start making contributions after 15 days because my exams are going on. Can you please give me a little tour of how things are going on in those projects so that I can get a hang of it in 15 days?